Kickstarter-iOS/Views/PledgeViewCTAContainerView.swift

```diff
@@ -0,0 +1,194 @@
+import KsApi
+import Library
+import PassKit
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Button {
+    static let minHeight: CGFloat = 48.0
+  }
+}
+
+final class PledgeViewCTAContainerView: UIView {
+  // MARK: - Properties
+
+  private lazy var applePayButton: PKPaymentButton = { PKPaymentButton() }()
+
+  private lazy var ctaStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var disclaimerLabel: UILabel = { UILabel(frame: .zero) }()
+
+  private lazy var disclaimerStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var pledgeCTAButton: UIButton = {
+    UIButton(type: .custom)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var rootStackView: UIStackView = {
+    UIStackView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.configureSubviews()
+    self.setupConstraints()
+    self.bindViewModel()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> \.layoutMargins .~ .init(all: Styles.grid(3))
+
+    _ = self.applePayButton
+      |> applePayButtonStyle
+
+    _ = self.ctaStackView
+      |> ctaStackViewStyle
+
+    _ = self.disclaimerLabel
+      |> disclaimerLabelStyle
+
+    _ = self.disclaimerStackView
+      |> disclaimerStackViewStyle
+
+    _ = self.layer
+      |> layerStyle
+
+    _ = self.pledgeCTAButton
+      |> greenButtonStyle
+      |> UIButton.lens.title(for: .normal) %~ { _ in Strings.Pledge() }
+
+    let isAccessibilityCategory = self.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+
+    _ = self.rootStackView
+      |> adaptableStackViewStyle(isAccessibilityCategory)
```

Comment:
This doesn't need to be adaptable because we're not changing the axis for accessibility sizes.